src/validation/fields/date.validation.ts

```diff
@@ -1,12 +1,13 @@
 import { body } from "express-validator";
 
-import { checkDate, checkDateIsInPast } from "../custom.validation";
+import { checkDate, checkDateIsInPast, checkDateValueIsValid } from "../custom.validation";
 import { ErrorMessages } from "../error.messages";
 
 export const start_date_validations = [
   body("start_date")
     .custom((value, { req }) => checkDate(ErrorMessages.MANAGING_OFFICER_START_DATE, req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"]))
-    .custom((value, { req }) => checkDateIsInPast(ErrorMessages.MO_START_DATE_NOT_IN_PAST, req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"])),
+    .custom((value, { req }) => checkDateIsInPast(ErrorMessages.MO_START_DATE_NOT_IN_PAST, req.body["start_date-day"], req.body["start_date-month"], req.body["start_date-year"]))
```

Comment:
Check is valid should be added before checking id it is in the pass

Reply:
The order will not block the checks for going forward so we will end up with different message errors. We should use the [oneOf](https://express-validator.github.io/docs/check-api.html#oneofvalidationchains-message) that comes with [express-validator](https://express-validator.github.io/docs/), but it is part of another story.

We will need to fix the `checkDateIsInPast` method and validate the values passed, I will do it in another PR. Thanks Matthew